packages/react-vapor/src/components/loading/components/TableLoading.tsx

```diff
@@ -78,17 +78,26 @@ const CardSubRow = ({num}: {num: number}) => (
 const TableRow = ({numberOfColumns, nColumn}: {numberOfColumns?: number; nColumn: number}) => (
     <tr className="mod-border-bottom no-hover">
         {_.times(numberOfColumns, (nRow: number) => (
-            <Row key={`table-row-loading-${nRow}`} num={nColumn} />
+            <Cell key={`table-row-loading-${nRow}`} num={nColumn} />
         ))}
     </tr>
 );
 
+/**
+ * @deprecated use Cell instead as \<Row /\> doesn't render a \<tr /\>
+ */
 const Row = ({num}: {num: number}) => (
     <td className="table-cell-loading">
         <div className={classNames('table-cell-content-loading', {'mod-half': num % 2})} />
     </td>
 );
```

Comment:
Avoid code duplication:
```suggestion
const Row = ({num}: {num: number}) => <Cell num={num} />;
```